### ISSUE TITLE
No longer boots to desktop

### ISSUE THREAD

**Issue by kultigspritzig** [syscl/XPS9350-macOS#89]
I don't know what happened. Haven't tinkered with it for few weeks, haven't even updated for a while. I restarted to Windows today and after coming back to Mac OS it boots like usual, with Mac logo, then progress bar, then the expected artifact when switching resolutions and just when I'd expect the desktop to show up, it doesn't. It's not hung up, though - if I run in verbose mode, I can see the greyed-out  terminal in the background and it reacts to keyboard. I tried single mode and running fsck, as well as safe mode. Nothing helps.

I don't know what could happen - I used it like a regular Mac for a while now, no upgrades, no tuning. The last thing I remember I installed was the update for XCode command line set, but I am pretty sure I restarted to Windows since then and back. Also, I'm still on 10.2.3.

I suspect this has something to do with the resolution it tries and fails to set, because screen blinks and shows the progress circle few times before it gives up.

EDIT: I removed the windowserver plist files. The resolution is clearly lower now, but still fails to boot. What the hell?!

**Comment by kultigspritzig** [comment]
![img_1349](https://cloud.githubusercontent.com/assets/25629364/24574939/444c0332-1668-11e7-8ae5-d573beaa1c5f.JPG)

The loginwindow issue seems to be the underlying cause. Anyone seen anything like that before?

**Comment by syscl** [comment]
seems CoreDisplay/IOKit issue, try to revert back to original ```/System/Library/Frameworks/CoreDisplay.framework/Versions/Current/CoreDisplay``` or ```/System/Library/Frameworks/IOKit.framework/Versions/Current/IOKit```, then boot with injectIntel = False, execute deploy.sh to re patch CoreDisplay/IOKit again.

syscl

**Comment by kultigspritzig** [comment]
This looks like something more than that. I am trying to boot the Installation USB - the same one I used for installation, since I don't have recovery HD partition. First time I tried it resterted spontaneously as soon as the circling color cursor appeared. Second time I managed to get further and opened the Disk Utility, fucked First Aid and it restarted again. Third time it restarted same moment the first time.

Windows works just fine, so I don't see how this could be a hardware issue? I will run some stress tests nonetheless to confirm.
